Source/Scene/PostProcessorScene.js

```diff
@@ -20,23 +20,247 @@ define([
      */
     function PostProcessorScene() {
         this.blackAndWhiteStage = createBlackAndWhiteStage();
+        this.blackAndWhiteStage.show = false;
         this.brightnessStage = createBrightnessStage();
+        this.brightnessStage.show = false;
         this.eightBitStage = createEightBitStage();
+        this.eightBitStage.show = false;
         this.compositeTextureStage = createCompositeTextureStage();
         this.compositeTextureStage.show = false;
+        this.LensFlareStage = createLensFlareStage();
 
         var stages = [
             this.blackAndWhiteStage,
             this.brightnessStage,
             this.eightBitStage,
-            this.compositeTextureStage
+            this.compositeTextureStage,
+            this.LensFlareStage
         ];
 
         this._postProcessor = new PostProcessor({
             stages : stages
         });
     }
 
+    function createLensFlareStage() {
+        var url_Dirt = buildModuleUrl('Assets/Textures/LensFlare/DirtMask.jpg');
+        var url_Star = buildModuleUrl('Assets/Textures/LensFlare/StarBurst.jpg');
+
+        var uniformValues = {
+
+            DirtTexture: url_Dirt,
+            StarTexture: url_Star,
+
+            Intensity: 2.0,
+
+            Distortion: 5.0,
+            NumofGhosts: 4,
+            GhostDispersal: 0.4,
+
+            HaloWidth: 0.4,
+
+            EarthRadius: 0.0
+        };
+
+        var LensFlareShader =
+            'uniform sampler2D u_colorTexture; \n' +
+            'uniform sampler2D u_DirtTexture; \n' +
+            'uniform sampler2D u_StarTexture; \n' +
+
+            'uniform float u_Distortion; \n' +
+            'uniform float u_GhostDispersal; \n' +
+            'uniform float u_HaloWidth; \n' +
+            'uniform float u_EarthRadius; \n' +
+
+            'uniform float u_Intensity; \n' +
+
+            'varying vec2 v_textureCoordinates; \n' +
+
+            //return ndc from world coordinate biased EarthRadius
+            'vec4 GetNDCFromWC(in vec3 WC, in float EarthRadius) \n' +
```

Comment:
Could you make this a built-in Cesium GLSL function including a unit test?

For example, see

https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Source/Shaders/Builtin/Functions

and

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Renderer/BuiltinFunctionsSpec.js

To learn more about unit tests in general, see our [Testing Guide](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide#testing-guide), especially the [GLSL section](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide#glsl).

Also rename this to `czm_WorldCoordinatestoNDC` like our color transform functions.